manim/default.cfg

```diff
@@ -147,6 +147,14 @@ streaming_console_banner = Manim is now running in streaming mode.
     >>> c = Circle()
     >>> manim.play(ShowCreation(c))
 
+[custom_folders]  # Syntax : custom_root folder e.g. *Videos* or *Documents/manim* and then the subfolders for the specific scene elements
+custom_root = Videos
```

Comment:
Issue: I am strongly against using a capitalized folder name as the default. I suggest we use `videos` instead.

Reply:
is this going to be a folder in relation to the cwd? If so, it should be lowercase. The capitalized folders should be only the OS-generated ones (at least considering the default behavior of the program) as some sort of convention